### PR TITLE
Update WM.yaml

### PR DESCRIPTION
**Pull request by RathishKJ**

### CODE REVIEW

File: ontology/yaml/resources/METERS/entity_types/WM.yaml [hunk 1]
```diff
@@ -28,4 +28,6 @@ WM_WF:
   - WM
   is_canonical: true
   uses:
-  - supply_water_flowrate_sensor
\ No newline at end of file
+  - supply_water_flowrate_sensor
```
Comment (tasodorff) [inline ontology/yaml/resources/METERS/entity_types/WM.yaml:31]: flowrate sensor, remove supply water

**Review by tasodorff — CHANGES_REQUESTED** [review]

File: ontology/yaml/resources/METERS/entity_types/WM.yaml (hunk 1 above)
Comment (RathishKJ) [reply ontology/yaml/resources/METERS/entity_types/WM.yaml:31]: This is an existing filed in the canonical present in the git.

Comment (RathishKJ) [reply ontology/yaml/resources/METERS/entity_types/WM.yaml:31]: > This is an existing filed in the canonical present in the git. Wouldn't it create conflict if we delete supply_water, elsewhere.

Comment (RathishKJ) [reply ontology/yaml/resources/METERS/entity_types/WM.yaml:31]: created a new canonical 

WM_WVFM:
  description: "Water volume accumulator and flow rate measurement ."
  implements:
  - WM
  is_canonical: true
  uses:
  - flowrate_sensor
  - water_volume_accumulator

**Comment by tasodorff** [comment]
failing validation

**Comment by RathishKJ** [comment]
> failing validation

The validation has been successful without conflicts.

**Comment by RathishKJ** [comment]
Pull request has been closed as we have considered WM_standard which has flowrate_sensor in the opt uses.

**Pull request closed without merging**